sdk/python/kfp/components/_data_passing.py

```diff
@@ -139,23 +139,24 @@ def get_canonical_type_for_type_struct(type_struct) -> str:
         return None
 
 
-def get_deserializer_code_for_type(typ) -> str:
-    try:
-        return type_name_to_deserializer.get(get_canonical_type_struct_for_type[typ], None)
-    except:
-        return None
+# Doesn't seem to be used anymore, can be removed?
+# def get_deserializer_code_for_type(typ) -> str:
```

Comment:
This method doesn't seem to be referenced anywhere, happy to add a `deprecated` warning or something, not sure if the convention is to deprecate or just straight remove or whether it's used somewhere I just didn't find the reference for!

Reply:
It doesn't look like this would be used by any user. So I think it's fine to just delete the code.